src/lib/backend/Memory.js

```diff
@@ -311,6 +313,18 @@ export default class Memory {
     }
 
     /**
+    * publish a message to a channel, clients can subsribe to this channel
+    * to receive messages
+    * @param {string} channel - channel to push the message
+    * @param {string} message - message to be pushed to the channel
+    * @return {object} - current instance
+    */
+    publish(channel, message) {
+        this.channels.emit('utapi', message);
```

Comment:
Also consider this PR to be changing state and not final yet as I am trying to come up with a sync-async solution for Integration Utapi tests.